app/controllers/courses_controller.rb

```diff
@@ -1,5 +1,5 @@
 class CoursesController < ApplicationController
   def index
-    # @courses = current_user.courses
+    @courses = current_user.courses.includes(:roles)
```

Comment:
Notice though that now you will be working with `Role` objects rather than `Course`.